features/welcome/components/WelcomePage.web.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
```

Comment:
As eslint puts it:

error  'React' must be in scope when using JSX  react/react-in-jsx-scope